### ISSUE TITLE
headers_remove in exim configure file

### ISSUE THREAD

**Issue by NikolayDachev** [avleen/vexim2#21]
There is a issue with this if/or condition missed or extra '}' '{' i cannot find it    

```
headers_remove = ${if or { { <{$spam_score_int}{1} } \
                            { <{$spam_score_int} {${lookup mysql{select users.sa_tag * 10 from users,domains \
                               where localpart = '${quote_mysql:$local_part}' \
                               and domain = '${quote_mysql:$domain}' \
                               and users.on_spamassassin = 1 \
                               and users.domain_id=domains.domain_id} {$value}fail} } } \
                             { eq {0}{${lookup mysql{select users.sa_tag * 10 from users,domains \
                               where localpart = '${quote_mysql:$local_part}' \
                               and domain = '${quote_mysql:$domain}' \
                               and users.on_spamassassin = 0 \
                               and users.domain_id=domains.domain_id} {$value}fail} } } \
                           } {X-Spam-Score:X-Spam-Report} }
```

**Comment by rimas-kudelis** [comment]
According to the aforementioned Exim bug report, their fix is now targeted at 4.86 and up. Somebody should check whether our config needs any changes to work with their fixed code, and report it here.

**Comment by rimas-kudelis** [comment]
So, this is fixed since Exim 4.86, but we do need to adjust our files similarly to Exim4U/src@5dc80032c92165aa4144908ba633b2d969bd8fff and Exim4U/src@504e74a3091aaee64e7b27a7abdde594a6d72bef.
I guess we should leave the syntax that works with 4.86 onwards as default, add a commented out alternative for pre-4.83, and add a comment suggesting to leave both syntaxes commented out if you are using 4.83 to 4.85 and cannot upgrade.
